### PR TITLE
feat(stripe): Add stripe extension

### PR DESCRIPTION
**Pull request by vymarkov**

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -1356,6 +1356,12 @@
       "version": "1.5.3",
       "checkout": "v1.5.3"
     },
+    {
+      "id": "stripe.vscode-stripe",
+      "repository": "https://github.com/stripe/vscode-stripe",
+      "version": "0.5.1",
+      "checkout": "v0.5.1"
```
Comment (jankeromnes) [inline extensions.json:1363]: On a side-note, it's generally safer and preferred to directly re-publish official `.vsix` releases (when available) as opposed to trying to re-package extensions ourselves (we might miss some steps and accidentally upload broken extensions to OpenVSX):

```suggestion
      "download": "https://github.com/stripe/vscode-stripe/releases/download/v0.5.1/vscode-stripe-0.5.1.vsix",
      "version": "0.5.1"
```

Anecdotally, I've also updated the `node add-extension` script to check for this and automatically use `.vsix` downloads whenever available.

**Review by jankeromnes — APPROVED** [review]
Looks good to me! 👍 

Many thanks @vymarkov 🙏

File: extensions.json (hunk 1 above)
Comment (vymarkov) [reply extensions.json:1363]: @jankeromnes yes, I've just reviewed new changes on master, that's why I converter to draft in order to add this extension by using new logic :)

**Review by jankeromnes — APPROVED** [review]
🚀

**Comment by jankeromnes** [comment]
Update: Auto-publishing worked! 🎉 https://open-vsx.org/extension/stripe/vscode-stripe

**Pull request merged**